### ISSUE TITLE
Unable to upload pkcs12 certificate - Could not parse certificate data - OpenSSL v3.0

### ISSUE THREAD

**Issue by FinnianDempsey** [OctopusDeploy/Issues#7156]
### Team

- [X] I've assigned a team label to this issue

### Severity

Customers not blocked as workaround exists. 2 Reported cases so far.

### Version

Confirmed on Latest - 2021.3.7082

### Latest Version

I could reproduce the problem in the latest build

### What happened?

When trying to upload a pkcs12 certificate to Octopus created using Openssl version 3.0 an error is received and the certificate is not uploaded.

### Reproduction

1. Create a pkcs12 using a command like:
`openssl pkcs12 -export -inkey key.pem -in cert.pem -out cert.pfx`
2. Upload the cert.pfx file to Octopus
3. See Error.

### Error and Stacktrace

```shell
Could not parse certificate data. Possible causes: 1) The certificate format is not supported. 2) The password is incorrect. 3) The file is corrupt. Error: Unable to parse certificate 'Test' (id: ): Unknown encryption algorithm: 1.2.840.113549.1.5.13
```

### More Information

This occurs whether a password is configured or not.

<img width="1430" alt="Screen Shot 2021-11-10 at 15 13 20" src="https://user-images.githubusercontent.com/78527975/141054065-7da78a7f-a780-4b7f-b964-edeed496f1f7.png">

[Internal Link - Zendesk](https://octopus.zendesk.com/agent/tickets/78825)
[Internal Link - Zendesk](https://octopus.zendesk.com/agent/tickets/78124)
[Internal Link - Slack](https://octopusdeploy.slack.com/archives/CNHBHV2BX/p1636521460258700)

### Workaround

Create a certificate using OpenSSL v1.1.1 instead

**Comment by patrick-smergut-octopus** [comment]
Another occurrence [Internal] - https://octopus.zendesk.com/agent/tickets/79604

**Comment by Clare-Octopus** [comment]
Another Occurance - https://octopus.zendesk.com/agent/tickets/79774

**Comment by SergeiDubovsky** [comment]
Another one: https://octopus.zendesk.com/agent/tickets/80543

**Comment by N-lson** [comment]
This was fixed in `2021.3.11057` - see https://github.com/OctopusDeploy/Issues/issues/7302
